config/config-example.js

```diff
@@ -157,6 +157,7 @@ exports.loguserstats = 1000 * 60 * 10; // 10 minutes
 // amount of traffic (i.e. hundreds of concurrent battles).
 exports.validatorprocesses = 1;
 exports.simulatorprocesses = 1;
+exports.verifierprocesses = 1;
```

Comment:
Can you just default to 1 if it's not in Config? I don't really want people to be fiddling with the verifier process count.